src/Components/ShoppingBill/ShoppingBill.jsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect } from "react";
+import axios from "axios";
+import "./ShoppingBill.css"
+import { useState } from "react"
+import { useCart, useToast } from "../../index"
+
+function ShoppingBill()
+{
+    const { userCart } = useCart()
+    const { showToast } = useToast()
+    let totalDiscount = 0, totalBill = 0;
+
+    userCart.forEach(product=>{
+        let discountOnCurrentProduct = ( (product.originalPrice - product.discountedPrice) * product.quantity )
+        console.log("Discount: ", discountOnCurrentProduct)
+        totalDiscount = totalDiscount + discountOnCurrentProduct
+        totalBill = totalBill + ( product.discountedPrice * product.quantity )
```

Comment:
are the brackets required here? isnt it by default * then +

Reply:
Yeah, it is for code readability